_episodes/21-carpentries.md

```diff
@@ -130,13 +152,22 @@ You may only call a workshop a Software Carpentry or Data Carpentry workshop if:
 *   you run our standardized pre- and post-workshop assessments and ensure everyone participates, and
 *   you send us summary information about attendees (at a minimum, the number of people who attended).
 
-### What's Core?
+### What is the Core Curriculum?
 
-A Software Carpentry workshop must include lessons on version control (e.g. git), the shell,
+A Software Carpentry workshop must include [lessons](https://software-carpentry.org/lessons/index.html) 
+on version control (e.g. Git), the UNIX shell,
 and a programming language (e.g. R or Python).
 
-A Data Carpentry workshop must include a Data Carpentry lesson on data organization and three
+A Data Carpentry workshop must include a Data Carpentry [lesson](https://datacarpentry.org/lessons/) 
+on data organization and three
 other modules in the same domain from the Data Carpentry curriculum.
+Published curricula include the Ecology, Genomics, Social Science and Geospatial workshop materials.
+Additionally, Astronomy, Digital humanities, Economics and Image analysis curriculum are under development. 
+
+A Library Carpentry [core curriculum](https://librarycarpentry.github.io/test/lessons/) 
```

Comment:
should read "A Library Carpentry workshop...."

Reply:
corrected!